docs/OpenTypeFeatureFileSpecification.md

```diff
@@ -310,6 +314,28 @@ the values of various table fields [§[9](#9)].
 
 _[ Note: Multiple master support has been withdrawn as of OpenType specification 1.3. ]_
 
+_[ Note: the following is unimplemented and is subject to change. ]_
+
+For **variable fonts only**, a `<metric>` value can make use of the following syntax to specify how values should vary based on axis locations:
+```
+(<location_spec1>:<value1> [...] <location_specN>:<valueN>)
+```
+
+Notes:
+ - A `<location_spec>` consists of one or more `<axis_tag>=<axis_location>` pairs, separated by commas 
+ - `<location_spec>:<value>` pairs are separated by spaces within the variable `<metric>`
+ - multiple `<location_spec>:<value>` pairs may be specified within a variable `<metric>`
+
+Example showing a simple single-axis variable scalar value. The value is -100 when Weight is 200, and -150 when Weight is 900:
+```
+<(wght=200:-100 wght=900:-150)>
+```
```

Comment:
It's unclear what happens when Weight is neither 200 or 900.

Reply:
@simoncozens could you comment/clarify here?

Reply:
Now I've been actually *using* this, I can add more here:

* We should probably clarify that values are in userspace coordinates.
* We should probably clarify that a value corresponding to the default location (the default axis value on each axis) must be included in the variable scalar.
* We may want to provide shortcut syntax for this, where the default location is elided, I don't know; e.g. `<(-100 wght=900:-150)>`

In answer to Fred's question, we could add something like "The value of the variable scalar at locations other than the provided master locations will be interpolated according to OpenType Font Variation interpolation rules" if necessary, but to me it feels tautological.